src/controllers/recipe.controller.ts

```diff
@@ -13,127 +13,125 @@ interface Pagination {
-      title,
-      description,
-      preparing,
-      ingredients,
-      userId: req.userId,
-      url,
-    };
+      const recipeData: Recipe = {
+        title,
+        description,
+        preparing,
+        ingredients,
+        userId: req.userId,
+        url,
+      };
 
-    const savedRecipe = await recipeService.createRecipe(recipeData);
-    return res.send(savedRecipe);
-  } catch (err) {
-    return res.status(500).send();
+      const savedRecipe = await recipeService.createRecipe(recipeData);
+      return res.send(savedRecipe);
+    } catch (err) {
+      return res.status(500).send();
+    }
   }
-}
 
-// reading recipes - if user is admin, then he sees all,
-// else user sees only recipes created by him
-// skip: number of items to skip, limit: how much to show
-// added filtering
-export async function getRecipeList(
-  req: Request,
-  res: Response
-): Promise<Response<RecipeDocument[]>> {
-  try {
-    const { userId, admin } = req;
-    const { skip, limit, name } = req.query;
-    let parsedName;
-    let recipes;
-    const parsedSkip = Number(skip);
-    const parsedLimit = Number(limit);
+  // reading recipes - if user is admin, then he sees all,
+  // else user sees only recipes created by him
+  // skip: number of items to skip, limit: how much to show
+  // added filtering
+  public async getRecipeList(
+    req: Request,
+    res: Response
+  ): Promise<Response<RecipeDocument[]>> {
+    try {
+      const { userId, admin } = req;
+      const { skip, limit, name } = req.query;
+      let parsedName;
+      let recipes;
+      const parsedSkip = Number(skip);
+      const parsedLimit = Number(limit);
 
-    if (name === undefined) {
-      parsedName = null;
-    } else {
-      parsedName = String(name);
-    }
+      if (name === undefined) {
+        parsedName = null;
+      } else {
+        parsedName = String(name);
+      }
 
-    if (admin) {
-      recipes = await recipeService.getRecipeList(
-        null,
-        parsedName,
-        parsedSkip,
-        parsedLimit
-      );
-    } else {
-      recipes = await recipeService.getRecipeList(
-        userId,
-        parsedName,
-        parsedSkip,
-        parsedLimit
-      );
+      if (admin) {
+        recipes = await recipeService.getRecipeList(
+          null,
+          parsedName,
+          parsedSkip,
+          parsedLimit
+        );
+      } else {
+        recipes = await recipeService.getRecipeList(
+          userId,
+          parsedName,
+          parsedSkip,
+          parsedLimit
+        );
+      }
+      return res.json(recipes);
+    } catch (err) {
+      return res.status(500).send();
     }
-    return res.json(recipes);
-  } catch (err) {
-    return res.status(500).send();
   }
-}
 
-// Reading recipe by id
-export async function readRecipeById(
-  req: Request,
-  res: Response
-): Promise<Response<RecipeDocument>> {
-  try {
-    const { id } = req.params;
+  // Reading recipe by id
```

Comment:
Comment not necessary. Unless we're writing detailed documentation, method name should be descriptive enough to tell us what it does. In this case we're just repeating ourselves